source/Patches/Roles/IVisualAlteration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TownOfUs.Patches.Roles.Modifiers
```

Comment:
```suggestion
﻿namespace TownOfUs
```
or maybe
```suggestion
namespace TownOfUs.Roles
```
since its not modifier specific

Reply:
Fair enough, I originally threw it in the Modifier folder before realizing I could generalize it, so that's what VS generated for namespace. Albeit the whole folder structure of this project is wonky already.